### DISCUSSION TITLE
MJpeg Marker

### DISCUSSION THREAD

**Discussion by LDwwh** [AxisCommunications/discussions#47] [ACAP]
Hello, I am parsing the binary MJpeg data from your AXIS F44 Dual Audio Input Main Unit, which is recored with format .mf4. 

I'd like to ask, what are the Markers for MJpeg data? Are they same with Jpeg (e.g. SOI, EOI described in [Wikipedia](https://en.wikipedia.org/wiki/JPEG) )?

Thanks a lot!
Dennis

**Comment by mattias-kindborg-at-work** [comment]
Hi @LDwwh. Sorry for the late response. Have you found an answer to your solution? Otherwise I will look into it.
